docs/overview/what_is__-__main_areas/con-application-connector-application-gateway.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Application Gateway
+type: Details
+---
+
+>**CAUTION:** This document describes the Application Gateway working in the alternative Compass [mode](#architecture-application-connector-components-application-operator). See the [architecture](#architecture-application-gateway) to learn about the Application Gateway working in the default legacy mode.
+
+The Application Gateway is an intermediary component between a Function or a service and an external API.
+
+## Proxying requests
+
+The Application Gateway proxies requests from Functions and services in Kyma to external APIs based on the configuration stored in Secrets.
+​
+### Proxy configuration
```

Comment:
I think it would be good to split the content of this file. I thought it's supposed to be a brief answer to the question "what is the Application Gateway", but then it suddenly provides several instructions how to configure it.

If the "what is...?" can be answered by just one line, we could collect those one-liners for all main areas in one common topic, and then this topic could be renamed along the lines of "how to configure the application gateway" instead of "what is...".